cmake/template/setup.bat.in

```diff
@@ -25,3 +25,8 @@ set "YARP_DATA_DIRS=%YARP_DATA_DIRS%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/shar
 rem Add the python bindings directory to the PYTHON_PATH
 set "PYTHONPATH=%PYTHONPATH%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/lib/python2.7/dist-packages"
 @endcmakeif ROBOTOLOGY_USES_PYTHON
+
+@cmakeif ROBOTOLOGY_USES_MATLAB
+rem Configure the Matlab
+set "MATLABPATH=%MATLABPATH%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/mex;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/share/WBToolbox;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%/share/WBToolbox/images"
```

Comment:
In the System Dependencies section we have:
> Note: tipically we assume that a user that selects the `ROBOTOLOGY_USES_MATLAB` also has Simulink installed in his computer.
If this is not the case, you can enable the advanced CMake option `ROBOTOLOGY_NOT_USE_SIMULINK` to compile all the subprojects that depend on MATLAB, but disable the subprojecs that depend on Simulink (i.e. the
[wb-toolbox](https://github.com/robotology/wb-toolbox) Simulink Library) if you have enabled the `ROBOTOLOGY_ENABLE_DYNAMICS` CMake options.

So why in the `robotology-superbuild/CMakeLists.txt` is the WB-Toolbox downloaded and compiled outside the `ROBOTOLOGY_USES_MATLAB` option flag. Shouldn't it be in the `if(ROBOTOLOGY_USES_MATLAB)...if(NOT ROBOTOLOGY_NOT_USE_SIMULINK)` section, along with the `whole-body-controllers`?